servers/rendering_server.h

```diff
@@ -1434,6 +1435,8 @@ class RenderingServer : public Object {
 	virtual void instance_set_surface_override_material(RID p_instance, int p_surface, RID p_material) = 0;
 	virtual void instance_set_visible(RID p_instance, bool p_visible) = 0;
 
+	virtual void instance_reset_interpolation(RID p_instance) = 0;
```

Comment:
Still not sure this name is good. With the `SceneTreeFTI`, this command will (currently anyway) no longer have any effect on the interpolation in the server.